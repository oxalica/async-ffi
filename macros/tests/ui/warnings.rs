#![deny(unused_variables)]
use async_ffi_macros::async_ffi;

#[async_ffi]
pub async fn unused_param(x: i32) -> i32 {
    42
}

fn main() {}
