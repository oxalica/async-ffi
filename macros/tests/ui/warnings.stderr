error: unused variable: `x`
 --> tests/ui/warnings.rs:5:27
  |
5 | pub async fn unused_param(x: i32) -> i32 {
  |                           ^ help: if this is intentional, prefix it with an underscore: `_x`
  |
note: the lint level is defined here
 --> tests/ui/warnings.rs:1:9
  |
1 | #![deny(unused_variables)]
  |         ^^^^^^^^^^^^^^^^
